Apps/phscreenshotmachine/ssmachine.json

```diff
@@ -30,6 +30,12 @@
             "description": "API Secret Phrase",
             "data_type": "password",
             "order": 1
+        },
+        "cache_limit": {
+            "description": "Cache Limit (how old cached images are accepted (in days))",
```

Comment:
```suggestion
            "description": "Maximum amount of time, in days, images can remain in the cache",
```

Reply:
Hello @rgil-splunk 
The API documentation for this app suggests that the image will always be kept in cache for 14 days and through this cache_limit parameter in the rest call a user can specify the limit on how old cached images are accepted.

Excerpts from the documentation: Repeated API calls (where all parameters are the same) are served from our image cache for 14 days. Using cacheLimit parameter, you can manage how old (in days) cached images do you accept.
Links: [https://www.screenshotmachine.com/website-screenshot-api.php](link1), [https://www.screenshotmachine.com/how-to.php](link2)

Please suggest if we need to make any changes.

Reply:
the wording sounds odd to me, but if it's fine to keep it so that it matches the official vendor documentation in that case